### PR TITLE
docs: update readme to include debugger

### PR DESCRIPTION
**Pull request by saihaj**
referencing ray debugger docs in the readme to make them more visible

CleanShot 2024-07-12 at 13 28 29@2x

### CODE REVIEW

**Review by angelinalg — COMMENTED** [review]
Let's be consistent with the order of listing:
1. Dashboard
2. Debugger

File: README.rst [hunk 1]
```diff
@@ -33,7 +33,10 @@ Or more about `Ray Core`_ and its key abstractions:
 - `Actors`_: Stateful worker processes created in the cluster.
 - `Objects`_: Immutable values accessible across the cluster.
 
-Monitor and debug Ray applications and clusters using the `Ray dashboard <https://docs.ray.io/en/latest/ray-core/ray-dashboard.html>`__.
+Learn more about Monitoring and Debugging:
+
+- Debug Ray applications with `Ray Distributed Debugger <https://docs.ray.io/en/latest/ray-observability/ray-distributed-debugger.html>`__.
```
Comment (angelinalg) [inline README.rst:38]:
```suggestion
- Debug Ray applications with the `Ray Distributed Debugger <https://docs.ray.io/en/latest/ray-observability/ray-distributed-debugger.html>`__.
```

File: README.rst [hunk 2]
```diff
@@ -33,7 +33,10 @@ Or more about `Ray Core`_ and its key abstractions:
...
+- Monitor Ray applications and clusters with `Ray dashboard <https://docs.ray.io/en/latest/ray-core/ray-dashboard.html>`__.
```
Comment (angelinalg) [inline README.rst:39]:
```suggestion
- Monitor Ray apps and clusters with the `Ray Dashboard <https://docs.ray.io/en/latest/ray-core/ray-dashboard.html>`__.
```

**Review by angelinalg — APPROVED** [review]

File: README.rst [hunk 3]
```diff
@@ -33,7 +33,10 @@ Or more about `Ray Core`_ and its key abstractions:
 - `Actors`_: Stateful worker processes created in the cluster.
 - `Objects`_: Immutable values accessible across the cluster.
 
-Monitor and debug Ray applications and clusters using the `Ray dashboard <https://docs.ray.io/en/latest/ray-core/ray-dashboard.html>`__.
+Learn more about Monitoring and Debugging:
+
+- Monitor Ray apps and clusters with the `Ray Dashboard <https://docs.ray.io/en/latest/ray-core/ray-dashboard.html>`__.
+- Debug Ray applications with the `Ray Distributed Debugger <https://docs.ray.io/en/latest/ray-observability/ray-distributed-debugger.html>`__.
```
Comment (saihaj) [inline README.rst:39]:
```suggestion
- Debug Ray apps with the `Ray Distributed Debugger <https://docs.ray.io/en/latest/ray-observability/ray-distributed-debugger.html>`__.
```

**Review by angelinalg — APPROVED** [review]

**Pull request merged**